src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4841indentation/InputFormattedClassWithChainedMethods.java

```diff
@@ -0,0 +1,28 @@
+package com.google.checkstyle.test.chapter4formatting.rule4841indentation;
+
+/** some javadoc. */
+public class InputFormattedClassWithChainedMethods {
+
+  public InputFormattedClassWithChainedMethods(Object... params) {}
```

Comment:
please rename original file to have Input prefix in name
ClassWithChainedMethods --> InputClassWithChainedMethods, same for ClassWithChainedMethodsCorrect

Reply:
done